src/ui/units/dash/containers/Dialogs/DialogRelations/components/Content/Row.tsx

```diff
@@ -162,24 +174,27 @@ const getConnectionByInfo = ({
     let field: ConnectionField = [];
     let type: ConnectionType = 'field';
 
-    if (hasFields) {
-        field = byFields;
-        type = 'field';
-    } else if (hasAliases) {
-        field = byAliases;
-        type = 'alias';
-    } else if (hasUsedParams) {
-        field = byUsedParams;
-        type = 'param';
+    if (availableLink) {
```

Comment:
If we will use `showByUsedParams` there `hasAliases` is not directly matching to  `availableLink`

Reply:
Got it. Yes it would be better here and in previous comment